.pipelines/pipeline.yaml

```diff
@@ -159,32 +158,18 @@ stages:
           name: "$(BUILD_POOL_NAME_LINUX_AMD64)"
         strategy:
           matrix:
-            cni_dropgz_linux_amd64:
+            cni_linux_amd64:
```

Comment:
I think this would require a similar change to the `.pipelines/cni/pipeline.yaml` as they use the dropgz test files